include/Driver.h

```diff
@@ -10,7 +10,9 @@
 #include <yaml-cpp/yaml.h>
 
 #include "cameras/CameraH264.h"
```

Comment:
Including camera headers is not necessary in Driver.h, because they are not in use there. Include in Driver.cpp?

Reply:
I usually keep all the includes in the header file if possible, and since it's done like that in the whole project I wouldn't change it.
If you prefer I can change that,